components/typography/styles.js

```diff
@@ -131,7 +134,7 @@ export const _generateBodySmallStyles = (selector) => {
 		@media (max-width: 615px) {
 			${selector} {
 				font-size: 0.6rem;
-				line-height: 0.9rem;
+				line-height: 0.9rem; /* TODO: what to do with this? */
```

Comment:
Still waiting on design to decide what to do with this responsive breakpoint. The `line-height` is now meaningless, but we may remove the entire rule.